### DISCUSSION TITLE
A leak rawFile in HarmonyOS libpag

### DISCUSSION THREAD

**Discussion by 1184893257** [Tencent/libpag#2575] [General] [answered]
<img width="937" alt="ec8b83958629a9b79b08145936db71b5" src="https://github.com/user-attachments/assets/ea477bcd-51db-4f80-95d8-9a89b76d11e7">

**Comment by Hparty** [comment, marked as answer]
https://github.com/Tencent/libpag/pull/2577 中已经修复，近期会更新新版本，感谢你发现的问题
